trigger_aws.yml

```diff
@@ -9,33 +8,35 @@ phases:
       # Device Farm support two major versions of Python, each with one minor version: 2 (minor version 2.7), and 3 (minor version 3.7.4).
       # The default Python version is 2, but you can switch to 3 by setting the following variable to 3:
       - export PYTHON_VERSION=3
-      
-      # By default, Appium server version used is 1.7.2.
-      # You can switch to an alternate supported version from 1.6.5, 1.7.1, 1.7.2, 1.8.0 or 1.8.1 by using a command like "avm 1.7.1"
-      # OR
-      # To install a newer version of Appium use the following commands:
+      # This command will install your dependencies and verify that they're using the proper versions that you specified in your requirements.txt file. Because Device Farm preconfigures your environment within a
+      # Python Virtual Environment in its setup phase, you can use pip to install any Python packages just as you would locally.
+      - cd $DEVICEFARM_TEST_PACKAGE_PATH
+      - . bin/activate
+      - pip install -r requirements.txt
+      # This test execution environment uses Appium version 1.9.1 by default, however we enable you to change it using the Appium version manager (avm). An
+      # example "avm" command below changes the version to 1.14.2.
+      # For your convenience, we have preinstalled the following versions: 1.9.1, 1.10.1, 1.11.1, 1.12.1, 1.13.0, 1.14.1, 1.14.2, 1.15.1 or 1.16.0.
+      # To use one of these Appium versions, change the version number in the "avm" command below to your desired version:
```

Comment:
I think there is no need to update these doc-strings.

Reply:
fixed